test/integration_test.js

```diff
@@ -779,10 +1059,10 @@ describe("Integration tests", () => {
             ]);
           });
 
-          it("should encode records when pushed to the server", () => {
```

Comment:
Yeah, using transofrmers end to end integration tests, we're pretty much safe here.